src/03-modality-agnostic-files.md

```diff
@@ -339,8 +339,16 @@ The purpose of this file is to describe timing and other properties of each
 imaging acquisition sequence (each *run* file) within one session.
 Each neural recording file should be described by at most one row.
 Relative paths to files should be used under a compulsory `filename` header.
-If acquisition time is included it should be under `acq_time` header.
+Exceptions are runs that consist of multiple parts.
+For example runs that include the `echo-`, `part-`, or `split-` fields.
```

Comment:
I don't understand how runs with multiple parts are an exception. It says above that each file gets a line, and when runs have multiple parts (i.e., multiple files), each file gets a line.

Reply:
I cannot make a code suggestion for this whole part because GitHub doesn't support it, but this is what I would write:

```Markdown
...
The purpose of this file is to describe timing and other properties of each
imaging acquisition sequence (each *run* file) within one session.

Each neural recording *file* SHOULD be described by one row.
Note that some neural recordings consist of multiple parts that are distributed over several files,
for example through `echo-`, `part-`, or `split-` entities.
Such recordings are still to be documented with one row per file, and
their acquisition time as documented under the `acq_time` column (see below)
MUST be identical, if that column is supplied.

Relative paths to files should be used under a compulsory `filename` header.

If acquisition time is included it should be listed under the `acq_time` header.
Acquisition time refers to when the first data point in each run was acquired.
Datetime should be expressed as described in [Units](./02-common-principles.md#units).
...
```

It seems to me like this part of the spec was written before the strict rules about "MUST" and "SHOULD" were introduced, so these words are used very leniently here ...

cc @tsalo @eort

Reply:
> It seems to me like this part of the spec was written before the strict rules about "MUST" and "SHOULD" were introduced, so these words are used very leniently here 

Should this be changed then? There are several lower case `should`s in the text. Are they different shoulds then the `SHOULD`'s or is that just inconsistent?

Reply:
@sappelhoff I like your version. But I think I'd prefer to mention the acq_time information for split files only after the acq_time information in general was described. So basically, not making this forward link (see below). But that's maybe style ;)

Reply:
> Such recordings are still to be documented

Does this imply MUST or SHOULD? (in the current commit I opted for MUST).

Reply:
> Should this be changed then? There are several lower case shoulds in the text. Are they different shoulds then the SHOULD's or is that just inconsistent?

The situation is unclear to me, let's not touch lines that we wouldn't have to touch anyways. But those that we *do* touch, we need to discuss

> Does this imply MUST or SHOULD? (in the current commit I opted for MUST).

"Such recordings are still to be documented"  implies a MUST :+1: 


> So basically, not making this forward link (see below). But that's maybe style preference ;)

I agree that that's a better style :+1: